docs/img/git-release-workflow.md

```diff
@@ -0,0 +1,65 @@
+```mermaid
```

Comment:
Add on the top of this markdown how did you get the svg out of it ... for the record :-)

Reply:
done